app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
```

Comment:
Will it work if I want to create an instance with `hidden = True`?

Reply:
Misunderstood task description , fixed =)